common/src/main/java/org/dash/wallet/common/util/GenericUtils.kt

```diff
@@ -48,56 +44,6 @@ object GenericUtils {
         }
     }
 
-    fun isInternetConnected(context: Context): Boolean {
-        val cm = context.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
-        return cm.activeNetworkInfo != null && cm.activeNetworkInfo!!.isConnected
-    }
-
-    /**
-     * Function which returns a concatenation of the currency code or currency symbol
-     * For currencies used by multiple countries, we set a locale with any country using the currency
-     * If the currentCurrencySymbol equals the currency code, we just use the currency code, otherwise we
-     * get the symbol
-     * @param currencyCode
-     * @return
-     */
-    fun setCurrentCurrencySymbolWithCode(currencyCode: String): String? {
-        var currentLocale = Locale("", "")
-        var currentCurrencySymbol: String? = ""
-        when (currencyCode.lowercase()) {
```

Comment:
Was this method unnecessary or handled by standard android functions?

Reply:
This was only used in the old `EnterAmountFragment` to figure out where the Dash symbol should go; we use `isCurrencyFirst` for it now which makes use of Androids `NumberFormat`.